content/it/docs/reference/glossary/image.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Image
+id: image
+date: 2022-07-30
+full_link: 
+short_description: >
+  Istanza archiviata di un cointainer che contiene un insieme di software e librerie necessarie per eseguire l'applicazione.
+
+aka: 
+tags:
+- fundamental
+---
+ Istanza archiviata di un {{< glossary_tooltip term_id="container" >}} che contiene un insieme di software e librerie necessarie per eseguire l'applicazione.
+
+<!--more-->
+
+Un modo di raccogliere assieme il software che permette di immagazzinarlo in un registro di container, scaricarlo in un sistema locale ed eseguirlo come un'applicazione. I metadati sono inclusi nell'immagine e possono contenere informazioni su come avviare l'esecuzione, chi ha prodotto l'immagine, o altro.
```

Comment:
```suggestion
Un modo di distribuire software che permette di immagazzinarlo in Image Registry (un registro di container images), scaricarlo in un sistema locale ed eseguirlo come un'applicazione. I metadati sono inclusi nell'immagine e possono contenere informazioni su come avviare l'esecuzione, chi ha prodotto l'immagine, o altro.
```
Some time ago we agreed that we should not literally translate terms of common usage in the Italian IT community, because the resulting translation is hard to understand.
In that cases, it is ok to provide the English work + a translation in parenthesis only the first time the work is encountered in a page